packages/react-composites/src/composites/ChatComposite/styles/Chat.styles.ts

```diff
@@ -97,16 +90,15 @@ export const messageThreadChatCompositeStyles: MessageThreadStyles = { root: { m
 /**
  * @private
  */
-export const typingIndicatorChatCompositeStyles: TypingIndicatorStylesProps = {
-  typingString: { maxWidth: MESSAGE_THREAD_WIDTH }
```

Comment:
Where you able to confirm that participants who have ludicrously long names have their names wrap correctly and not cause scrollbars or anything?

Reply:
Good point. Let me test that. \m/

Reply:
Yes, it's wrapping properly :)